core/src/epicli/data/common/ansible/playbooks/roles/helm/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+apache_epirepo_path: "/var/www/html/epirepo"
+helm_version: v3.2.0
```

Comment:
Shouldn't this be in data/common/defaults? Version can be changed in the future, also I think that somebody would like to override path also.